app/javascript/dashboard/components/widgets/conversation/ConversationAdvancedFilter.vue

```diff
@@ -0,0 +1,266 @@
+    if (this.getAppliedFilters.length) {
+      this.appliedFilters = this.getAppliedFilters;
+    } else {
+      this.appliedFilters.push({
+        attribute_key: 'status',
+        filter_operator: 'equal_to',
+        values: '',
+        query_operator: 'and',
+      });
+    }
+  },
+  methods: {
+    getInputType(key) {
+      const type = this.filterTypes.find(filter => filter.attributeKey === key);
+      return type.inputType;
+    },
+    getOperators(key) {
+      const type = this.filterTypes.find(filter => filter.attributeKey === key);
+      return type.filterOperators;
+    },
+    // eslint-disable-next-line consistent-return
+    getDropdownValues(type) {
+      switch (type) {
+        case 'status':
+          return [
+            ...this.$t('CHAT_LIST.CHAT_STATUS_ITEMS').map(status => {
+              return {
+                id: status.VALUE,
+                name: status.TEXT,
+              };
+            }),
+            {
+              id: 'all',
+              name: this.$t('CHAT_LIST.FILTER_ALL'),
+            },
+          ];
+        case 'assignee_id':
+          return this.$store.getters['agents/getAgents'];
+        case 'contact':
+          return this.$store.getters['contacts/getContacts'];
+        case 'inbox':
+          return this.$store.getters['inboxes/getInboxes'];
+        case 'team_id':
+          return this.$store.getters['teams/getTeams'];
+        case 'campaign_id':
+          return this.$store.getters['campaigns/getAllCampaigns'].map(i => {
+            return {
+              id: i.id,
+              name: i.title,
+            };
+          });
+        case 'labels':
+          return this.$store.getters['labels/getLabels'].map(i => {
+            return {
+              id: i.id,
+              name: i.title,
+            };
+          });
+        case 'browser_language':
+          return languages;
+        case 'country_code':
+          return countries;
+        default:
+          break;
+      }
+    },
+    appendNewFilter() {
+      this.appliedFilters.push({
+        attribute_key: 'status',
+        filter_operator: 'equal_to',
+        values: '',
+        query_operator: 'and',
+      });
+    },
+    removeFilter(index) {
+      if (this.appliedFilters.length <= 1) {
+        this.showAlert(this.$t('FILTER.FILTER_DELETE_ERROR'));
+      } else {
+        this.appliedFilters.splice(index, 1);
+      }
+    },
+    submitFilterQuery() {
+      this.$v.$touch();
+      if (this.$v.$invalid) return;
+      this.$store.dispatch(
+        'setConversationFilters',
+        JSON.parse(JSON.stringify(this.appliedFilters))
+      );
+      this.appliedFilters[this.appliedFilters.length - 1].query_operator = null;
+      this.$emit('applyFilter', this.appliedFilters);
+    },
+    clearPreviousValues(index) {
+      this.appliedFilters[index].values = '';
+    },
+  },
+};
+</script>
+
+<style lang="scss">
+@import '~widget/assets/scss/variables.scss';
```

Comment:
This is supposed to be used in widget pack only. Please remove this and use CSS variables from shared/assets/stylesheets/

Reply:
Resolved in https://github.com/chatwoot/chatwoot/pull/3239/commits/84c44831e64e65e2efc240708bb12ceea26e22c2